### ISSUE TITLE
Does your implementation require activation in the final layers?

### ISSUE THREAD

**Issue by sarmientoj24** [mlyg/mixed-focal-loss#17]

**Comment by sarmientoj24** [comment]
does it work with logits or no?

**Comment by mlyg** [comment]
Hi sarmientoj24,

Yes, the way the loss functions are written assume the model has a **softmax** activation as its final layer.

Best,
Michael
